### PR TITLE
Create Netscaler_parser.csl

### PR DESCRIPTION
**Pull request by happy-jo**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by happy-jo** [comment]
@ashwin-patil are you looking for the syslog message from ALA or the raw syslog message from the syslog collector?

**Comment by ashwin-patil** [comment]
> @ashwin-patil are you looking for the syslog message from ALA or the raw syslog message from the syslog collector?

Thanks @happy-jo . 
Output from ALA with Syslogmessage column will be sufficient to test since KQL parser is extracting from this column.

**Comment by happy-jo** [comment]
@ashwin-patil please see this [file](https://github.com/happy-jo/crap/blob/master/query_data%20(1).csv).

File: Parsers/Netscaler_parser.csl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    CitrixApp_ProcId = extract(@"app_process_id\s(\d+)\s\-", 1, CitrixEventMessage),
+    CitrixApp_LaunchTime = extract(@"app_Launch_time\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixApp_Guid = extract(@"session_guid\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixApp_DeviceSN = extract(@"device_serial_number\s(\d+)\s\-", 1, CitrixEventMessage),
+    CitrixSrvMon_IP = extract(@"MonServiceBinding_(.*?)\:", 1, CitrixEventMessage),
+    CitrixDvcMon_IP = extract(@"server_serviceGroup_NSSVC_HTTP_(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_SrcIP = extract(@"Source\s(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_SrcPort = extract(@"Source\s{\d+[.]\d+[.]\d+[.]\d+}\:(.*?)\s", 1, CitrixEventMessage),
+    CitrixICA_DestIP = extract(@"Destionation\s(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_DestPort = extract(@"Destionation\s{\d+[.]\d+[.]\d+[.]\d+}\:(.*?)\s", 1, CitrixEventMessage),
```
Comment (ashwin-patil) [inline Parsers/Netscaler_parser.csl:61]: This looks like typo for **_Destination_**. Can you check for line 60 and 61 ?

Comment (happy-jo) [reply Parsers/Netscaler_parser.csl:61]: Good Catch, correction made.

**Pull request merged**